api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -447,6 +448,22 @@ public void updateDbRdrExport(RdrEntity entity, List<Long> idList) {
     rdrExportDao.save(exportList);
   }
 
+  /**
+   * Creates RdrWorkspaceCreator object that contains workspace creator information like id, given
+   * name and lastName
+   *
+   * @param creatorUser
+   * @return RdrWorkspaceCreator object
+   */
+  RdrWorkspaceCreator toRdrWorkspaceCreator(DbUser creatorUser) {
```

Comment:
Please make a new MapStruct mapper for this. There's a guide and tutorial in the top level README.